utils/onnx2json.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
Is it used systematically in the pipelines, or is it just a tool?
Is the goal to not have .onnx models in the git, and transform them to json instead? Please provide a bit of context to this.

Reply:
it's just a tool, just for the purpose of constructing parse lit tests

I clarified the purpose in the script's block comment